BuberBreakfast/Pages/Shop/View.cshtml

```diff
@@ -0,0 +1,44 @@
+@page
+@model BuberBreakfast.Pages.Shops.ShopPageModel
```

Comment:
# important
must be exactly the same as https://github.com/danielkwok21/learnDotNet6/pull/7/files#diff-50e996ab9c28814feb65bed43ea76fd8521aaf9680efb2bfe5c332a569db76c0R6

![image](https://github.com/danielkwok21/learnDotNet6/assets/28213363/bbab0c08-1f9f-4840-ae3d-fec5b80fb312)